eos-factory-reset.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Endless factory reset service
+ConditionPathExists=/var/.eos-factory-reset
+DefaultDependencies=no
```

Comment:
When using `DefaultDependencies=no` you need to explicitly add `Conflicts=shutdown.target`.